### ISSUE TITLE
Question about Language model task

### ISSUE THREAD

**Issue by iamxiaoyubei** [tensorflow/lingvo#117]
I would like to ask, which paper is your reference to the LM task? Also, is your LM loss function log perplexity? Is label the next word of input in LM task?

**Comment by jonathanasdf** [comment]
It seems it is based off of https://github.com/rafaljozefowicz/lm, which is based off of https://arxiv.org/abs/1602.02410.

**Comment by iamxiaoyubei** [comment]
Thank you~

**Comment by zh794390558** [comment]
watch this.
